examples/light-switch-app/silabs/src/BindingHandler.cpp

```diff
@@ -97,6 +133,9 @@ void LightSwitchChangedHandler(const EmberBindingTableEntry & binding, Operation
         case Clusters::OnOff::Id:
             ProcessOnOffGroupBindingCommand(data->commandId, binding);
             break;
+        case Clusters::LevelControl::Id:
+            ProcessLevelControlGroupBindingCommand(data->commandId, binding, data);
```

Comment:
See previous comment - mixing the application data to the binding data isn't a good way to go when we have multiple commands that require extra information.